feature-libs/product-configurator/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "@spartacus/product-configurator",
+  "version": "3.0.0-next.6",
+  "description": "Product configurator feature library for Spartacus",
+  "homepage": "https://github.com/SAP/spartacus",
+  "keywords": [
+    "spartacus",
+    "framework",
+    "storefront",
+    "product configurator"
+  ],
+  "scripts": {
+    "prepare:schematics": "yarn install",
```

Comment:
We no longer have any local dependencies in feature libraries. We use deps from the top level `node_modules`. You can get reference scripts from `feature-libs/storefinder`.

This will also mean you can drop `devDependecies` from this package.json and yarn.lock as well.

Reply:
Have changed that